spec/controllers/status_controller_spec.rb

```diff
@@ -61,7 +68,7 @@
     end
     describe 'with reindexing' do
       before :each do
-        expect(Dor::Item).to receive(:find).with(@druid).and_return(@item)
+        expect(Dor).to receive(:find).with(@druid).and_return(@item)
   #     expect(Dor::Item).to receive(:find).with(@druid).and_call_original
         expect(@item).to receive(:save)
       end
```

Comment:
This spec was driving me nuts with inconsistent behavior and a failure in Dor.find to handle an unusual Fedora exception.  The revisions to `find_druid` that intercept anything that's not a valid DRUID before calling Dor.find have resolved the spec failures on this one.